models/modelsmodule.h

```diff
@@ -49,6 +51,10 @@ class ModelsModule : public SLIModule
 
   const std::string name( void ) const;
   const std::string commandstring( void ) const;
+
+protected:
+  template < typename ConnectionT >
+  void register_connection_models( std::string name_postfix = "" );
```

Comment:
Please add Doxygen documentation for the method. 
And is this method specific to modelsmodule? Wouldn't it be useful also for other modules defining synapses and should thus rather be provided by `ModelManager`?

Reply:
- As it is a private method, add `_` at the end of the method name.
- Concerning the docstring:
    - No need to write that it is for internal use, the fact it is `private` makes that clear.
    - Docstring should start with a single line short description
    - Add a Note to make clear that for any new connection model, registration must be added in this method.